internals/daemon/daemon_test.go

```diff
@@ -60,6 +61,11 @@ type daemonSuite struct {
 var _ = Suite(&daemonSuite{})
 
 func (s *daemonSuite) SetUpTest(c *C) {
+	err := reaper.Start()
```

Comment:
I believe two instances of ```testutil.FakeCommand(c, "shutdown", "", false)``` with ```withReaper=false``` exist in the test code. Its only related to running shellcheck on the script, but I guess now that the reaper is getting the sigchilds, we should use the reaper command to start the shellchecks ?

Reply:
Yeah, good point. I can't stand that `shellcheck` stuff inside `go test`, particularly because it's "maybe shell check" -- it may or may not run depending on whether you've got `shellcheck` installed. IMO this kind of thing should be done in extended CI tests or not at all. I could refactor this stuff, but I'd prefer to just rip out the shellcheck bit. I'll open a separate PR for that shortly.